### PR TITLE
docs: initial governance draft

### PR DESCRIPTION
**Pull request by Josumner**

### CODE REVIEW

File: docs/docs/cli-governance.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Create initial proposal by uploading the JSON file
+# this is signed by a key 'proposer' that provides a portion of the current threshold deposit
+fetchd tx gov submit-proposal --proposal ~/json_path/proposal.json --deposit <deposit_value> --from proposer
+
+# In order to later refer to this proposal, the proposal_id can be determined
+fetchd query gov proposals
+```
+
+### Proposal voting and querying
+After the deposit period has passed, there are two outcomes: either the current minimum threshold is met, or the value is not met and the funds are burned. In the first case this proposal is submitted and then voted on, returning a tally at the end of the voting period.
```
Comment (daeMOn63) [inline docs/docs/cli-governance.md:47]: `"the funds are burned" ` Wow this sounds scary. Is there any reference we could link here or add some description on how to define an appropriate amount for the proposal value ?

File: docs/docs/cli-governance.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+At any point of the deposit stage, the deposit value can be queried.
+```
+# This command returns a text representation of the current total deposit value of a proposal
+fetchd query gov deposits <proposal_id>
+
+# Other users may contribute to funding the proposal using
+fetchd tx gov deposit <proposal_id> <deposit_amount> --from contributer
+```
+
+In order to submit a vote on a proposal that has passed into the voting phase, all users except the proposer may do so using this command.
```
Comment (daeMOn63) [inline docs/docs/cli-governance.md:58]: `all users except the proposer can vote`: is all users everyone or just the ones that added their share to the global deposit ?

File: docs/docs/cli-governance.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```
+# Submit a vote from a key 'voter' with the desired outcome of the voter
+fetchd tx gov vote <proposal_id> <yes|no|no_with_veto|abstain> --from voter
+```
+
+After this deposit phase, the current voting turnout and tally can be queried, which displays a list of all voters and their choice.
```
Comment (daeMOn63) [inline docs/docs/cli-governance.md:64]: `displays a list of all voters`: what do you think adding some sample output matching your demo proposal above?

**Review by daeMOn63 — COMMENTED** [review]
looks good! just have some general questions, haven't had time to play with the gov module yet so I'm curious :)

File: docs/docs/cli-governance.md (hunk 2 above)
Comment (Josumner) [reply docs/docs/cli-governance.md:58]: I was under the impression that after the deposit is met, anyone can vote on the issue - but I could be wrong

File: docs/docs/cli-governance.md (hunk 3 above)
Comment (Josumner) [reply docs/docs/cli-governance.md:64]: It's a bit of a clunky printout, a list of each voter and their vote, but I could definitely provide a sample of a few

**Review by daeMOn63 — APPROVED** [review]

File: docs/docs/cli-governance.md (hunk 1 above)
Comment (daeMOn63) [reply docs/docs/cli-governance.md]: Right so the funds aren't burned in all cases. From the link, you added (https://docs.cosmos.network/master/modules/gov/01_concepts.html#deposit-refund-and-burn), that burn only happens if the proposal is vetoed by a majority. Accepted or rejected proposals get their deposit refunded, which is a lot less scary. @Josumner

**Review by daeMOn63 — APPROVED** [review]

**Pull request merged**